app/controllers/histogram_chart_controller.rb

```diff
@@ -1,10 +1,11 @@
 require 'histogram/array'  # enables Array#histogram
 require 'googlecharts'
```

Comment:
I had to comment these out to make the histogram page work on linux, are you sure its required?